extension/intents/tabs/tabs.toml

```diff
@@ -226,3 +226,12 @@ match = """
   (Go | switch | change | activate |) (to |) (the |) first (pinned | app) tab [dir=first]
   (Go | switch | change | activate |) (to |) (the |) last (pinned | app) tab  [dir=last]
 """
+
+[tabs.moveSelectedToNewWindow]
+description = "Move selected tabs to (a |) new window"
+match = """
+  move selected tabs to new window
+"""
```

Comment:
Oops, you put the match pattern in `description` and the description in `match`

Some other phrases:

* `(launch | start |) new window (with |) selected tab{s}`
* `(move | launch | start) selected tabs (to | in |) (new |) window`

I guess that's all I can think of for now...